### PR TITLE
feat: company-section

### PR DESCRIPTION
**Pull request by Pratham-Mishra178**

### CODE REVIEW

**Comment by ashuvssut** [comment]
@Pratham-Mishra178 I think you did something wrong in following the step

I can see commit `b60ba37` (created by me) in this PR.

There must be some mistake. lets discuss in meet. DM me when you will be free

File: pages/company/company.module.scss [hunk 1]
```diff
@@ -0,0 +1,187 @@
+main {
```
Comment (ashuvssut) [inline pages/company/company.module.scss:1]: I think you copied the whole code form some other `.scss` file

remove the scss code which is not required for company page

File: pages/company/components/Story.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-unescaped-entities */
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { FC } from "react";
+import styles from "../company.module.scss";
+import { story4 } from "assets/png";
```
Comment (ashuvssut) [inline pages/company/components/Story.tsx:6]: (rename story4)

File: pages/company/components/Story.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	{ title: "	✓ $33M in funding" },
+	{ title: "	✓ 500+ global meetups" },
+];
+export const Story: FC = () => {
+	return (
+		<div className={styles.story}>
+			<Typography
+				variant="h4"
+				sx={theme => ({
+					[theme.breakpoints.down("sm")]: {
```
Comment (ashuvssut) [inline pages/company/components/Story.tsx:21]: change breakpoint to center the text in tablet made

Comment (ashuvssut) [reply pages/company/components/Story.tsx:21]: better use `sx={ { md : {textAlign: "center"} } }` ==> this one looks short and simpler

File: pages/company/components/Story.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+					cupiditate iste, asperiores totam exercitationem?{" "}
+				</Typography>{" "}
+				<br />
+				<Box
+					className={styles.cards}
+					sx={theme => ({
+						[theme.breakpoints.down("sm")]: {
+							gridTemplateAreas: '"a" "b" "c" "d" !important',
+							width: "100%",
+						},
```
Comment (ashuvssut) [inline pages/company/components/Story.tsx:44]: I think this sx is unnecessary. Its copied from other .tsx

@Asmi-1 You did the same mistake na. tell pratham all the required edits (everything that I told you to do in your PR => you may show him your final component).

File: pages/company/components/Story.tsx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+					})}
+				>
+					{story.map((feat, i) => {
+						const textAlign = i % 2 === 0 ? "left" : "right";
```
Comment (ashuvssut) [inline pages/company/components/Story.tsx:48]: @Asmi-1 same here too... `textAlign`...

File: pages/company/components/Story.tsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+								})}
+							>
+								<Typography variant="subtitle1">{feat.title}</Typography>
+							</Box>
+						);
+					})}
+				</Box>
+			</Box>
+			<Box mt={5} display="flex">
+				<div>
```
Comment (ashuvssut) [inline pages/company/components/Story.tsx:68]: unnecessary div.

File: pages/company/components/Story.tsx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+					<Image src={story4} alt="" />
```
Comment (ashuvssut) [inline pages/company/components/Story.tsx:69]: image

1. The whole parent box is not responsive for smaller screens. so apply these styles:
- first put `max-width: 600px`to this image
- add `flex-wrap: wrap` to tha parent

2. add margins to the text. (its touching the boundaries)

Comment (ashuvssut) [reply pages/company/components/Story.tsx:69]: Do all these changes... rest LGTM

**Review by ashuvssut — CHANGES_REQUESTED** [review]

File: pages/company/components/Story.tsx (hunk 7 above)
Comment (Pratham-Mishra178) [reply pages/company/components/Story.tsx:69]: how to give max-width to the image

Comment (ashuvssut) [reply pages/company/components/Story.tsx]: You cant.

For that I earlier created our own `Image` component. That custom `Image` component's `wrapstyle` prop add these kind of styles. (Look at our custom `Image` component code.. its a global component. You will get an idea)

File: pages/company/components/Story.tsx [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+					quibusdam voluptates dolores pariatur porro ipsum nisi illo,
+					cupiditate iste, asperiores totam exercitationem?{" "}
+				</Typography>{" "}
+				<br />
+				<Box
+					className={styles.cards}
+					gridTemplateAreas={{ xs: '"a" "b" "c" "d"', sm: '"a b" "c d"' }}
+				>
+					{story.map((feat, i) => {
+						const textAlign = i % 2 === 0 ? "left" : "right";
```
Comment (ashuvssut) [inline pages/company/components/Story.tsx:44]: where is this `textAlign` getting used?

File: pages/company/company.module.scss [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+		display: flex;
+		flex-wrap: wrap;
+		padding: 4vw;
+		gap: 30px;
+		justify-content: space-around;
+		align-items: center;
+
+		.cards {
+			display: grid;
+			grid-template-areas:
```
Comment (ashuvssut) [inline pages/company/company.module.scss:27]: I dont think this `grid-template-areas` is needed anymore cuz you added this same CSS property through  `gridTempleAreas` prop in the `Box` component (Story.tsx line 41)

File: pages/company/company.module.scss [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+				"a b"
+				"c d";
```
Comment (ashuvssut) [inline pages/company/company.module.scss:29]: I dont think this `grid-template-areas` is needed anymore cuz you added this same CSS property through `gridTempleAreas` prop in the `Box` component (Story.tsx line 41)

**Review by ashuvssut — CHANGES_REQUESTED** [review]

File: pages/company/components/Story.tsx (hunk 8 above)
Comment (Pratham-Mishra178) [reply pages/company/components/Story.tsx:44]: in the mobile view the texts were aligning themselves alternatively left and right.

File: pages/company/company.module.scss (hunk 10 above)
Comment (Pratham-Mishra178) [reply pages/company/company.module.scss:29]: yes I have updated that just forget to ammend

File: pages/company/components/Story.tsx (hunk 8 above)
Comment (ashuvssut) [reply pages/company/components/Story.tsx:44]: yes, but `textAlign` is not getting used anywhere. I think its redundant cuz its not getting used

Comment (Pratham-Mishra178) [reply pages/company/components/Story.tsx:44]: ohk i will remove this

File: pages/company/company.module.scss (hunk 1 above)
Comment (ashuvssut) [reply pages/company/company.module.scss:1]: Is this resolved?

Comment (Pratham-Mishra178) [reply pages/company/company.module.scss]: yes

File: pages/company/company.module.scss (hunk 10 above)
Comment (Pratham-Mishra178) [reply pages/company/company.module.scss]: resolved

File: pages/company/components/Story.tsx (hunk 8 above)
Comment (Pratham-Mishra178) [reply pages/company/components/Story.tsx]: resolved

**Review by ashuvssut — APPROVED** [review]
@Pratham-Mishra178 i removed all the remaining redundant copied code this time. make sure next time you understand which part of code actually needs to be copied

**Comment by ashuvssut** [comment]
image
for your next pr do these icons properly. use a single icon component instead of using ASCII letter 4 times

**Comment by ashuvssut** [comment]
image
for your next pr do these icons properly. use a single icon component in the `map` func instead of using ASCII letter 4 times

**Comment by Pratham-Mishra178** [comment]
Okay

**Pull request merged**